build_scripts/windows/scripts/az_spython.cmd

```diff
@@ -0,0 +1,12 @@
+::
+:: Microsoft Azure CLI - Windows Installer - Author file components script
+:: Copyright (C) Microsoft Corporation. All Rights Reserved.
+::
+@IF EXIST "%~dp0\..\python.exe" (
+  SET AZ_INSTALLER=MSI
+  SET IS_SPYTHON=true
+  "%~dp0\..\python.exe" "%~dp0\..\Lib\site-packages\azure\cli\__main__.pyc" %*
```

Comment:
Spython disable `python -m` when WDAC is enabled. Run pyc file directly. https://dev.azure.com/mseng/Python/_git/spython?path=/doc/overview.md&_a=preview&anchor=restricted-entry-point